diffpy/snmf/factorizers.py

```diff
@@ -27,7 +28,7 @@ def lsqnonneg(stretched_component_matrix, target_signal):
     Raises
```

Comment:
please see the comment in the `mkr_box` PR.  Delete this and use the discussion from there.

Reply:
I will delete it